src/docs/doxygen/Doxyfile.in

```diff
@@ -1012,7 +1012,7 @@ STRIP_CODE_COMMENTS    = NO
 # function all documented functions referencing it will be listed.
 # The default value is: NO.
 
-REFERENCED_BY_RELATION = YES
+REFERENCED_BY_RELATION = NO
```

Comment:
This creates a lot of excess information on the doxygen functions.  IMO it clutters the page.  I can undo this change if I am the only one that thinks this way.  Example:

https://axom.readthedocs.io/en/develop/doxygen/html/classaxom_1_1Array.html#a6f293178b89b281da4c08579e309d7ee

vs:

https://axom.readthedocs.io/en/feature-white238-slic_root/doxygen/html/classaxom_1_1Array.html#a6f293178b89b281da4c08579e309d7ee

Reply:
I agree that the info is excessive and not helpful.

Reply:
I'll third the vote to disable this.